pkg/controllers/disruption/orchestration/queue.go

```diff
@@ -202,10 +204,11 @@ func (q *Queue) Reconcile(ctx context.Context, _ reconcile.Request) (reconcile.R
 		// Log the error
 		logging.FromContext(ctx).With("nodes", strings.Join(lo.Map(cmd.candidates, func(s *state.StateNode, _ int) string {
 			return s.Name()
-		}), ",")).Errorf("failed to disrupt nodes, %s", multiErr)
+		}), ","), "command-id", cmd.id).Errorf("failed to disrupt nodes, %s", multiErr)
```

Comment:
Add this command-id to the context by injecting it into the logger at the top of the reconciliation loop?

Reply:
Might make it easier so that we don't forget adding it to each call that we have to log